packages/foundation/tools/eslint-config/index.js

```diff
@@ -186,6 +187,27 @@ export default tseslint.config(
       'jest/valid-title': 'off',
     },
   },
+  {
+    files: ['**/*.spec.ts'],
+    plugins: {
+      vitest,
+    },
+    rules: {
+      ...vitest.configs.recommended.rules,
+      ...vitest.configs.all.rules,
+      '@typescript-eslint/unbound-method': 'off',
+      '@typescript-eslint/no-confusing-void-expression': 'off',
+      '@typescript-eslint/no-empty-function': 'off',
```

Comment:
Why is this rule disabled?

Reply:
Sorry, I just copied from another project, I can't remember it xD. I will remove it